ui/app/src/components/PagesWidget.tsx

```diff
@@ -0,0 +1,692 @@
+  const classes = useStyles({});
+  const { item, onItemSelected, currentLocale, selectMode, onSelectItem, onOpenItemMenu } = props;
+  const [over, setOver] = useState(false);
+  return (
+    <ListItem
+      className={clsx(classes.pagesNavItem, selectMode && 'noLeftPadding')}
+      onMouseEnter={() => setOver(true)}
+      onMouseLeave={() => setOver(false)}
+    >
+      {
+        selectMode &&
+        <Checkbox
+          color="default"
+          className={classes.pagesNavItemCheckbox}
+          onChange={(e) => {
+            onSelectItem(item, e.currentTarget.checked)
+          }}
+          value="primary"
+        />
+      }
+      <Typography
+        variant="body2"
+        className={clsx(classes.pagesNavItemText, (currentLocale !== item.localeCode) && 'opacity')}
+      >
+        {item.label}
+        {
+          (currentLocale !== item.localeCode) &&
+          <FlagRoundedIcon className={classes.flag}/>
+        }
+      </Typography>
+      {
+        over &&
+        <div className={classes.optionsWrapper}>
+          <IconButton aria-label="options" className={classes.icon}
+                      onClick={(event) => onOpenItemMenu(event.currentTarget)}>
+            <MoreVertIcon/>
+          </IconButton>
+          <IconButton aria-label="options" className={classes.icon} onClick={() => onItemSelected(item)}>
+            <ChevronRightRoundedIcon/>
+          </IconButton>
+        </div>
+      }
+    </ListItem>
+  )
+}
+
+interface PagesNavProps {
+  items: Item[];
+  currentLocale: string;
+  selectMode: boolean;
+
+  onSelectItem(item: Item, unselect: boolean): void;
+
+  onItemSelected(item: Item): void;
+
+  onOpenItemMenu(element: Element): void;
+}
+
+function PagesNav(props: PagesNavProps) {
+  const { items, onItemSelected, currentLocale, selectMode, onSelectItem, onOpenItemMenu } = props;
+  return (
+    <List component="nav" aria-label="pages nav" disablePadding={true}>
+      {
+        items.map((item: Item) =>
+          <PagesNavItem
+            item={item}
+            key={item.id}
+            onItemSelected={onItemSelected}
+            currentLocale={currentLocale}
+            selectMode={selectMode}
+            onSelectItem={onSelectItem}
+            onOpenItemMenu={onOpenItemMenu}
+          />
+        )
+      }
+    </List>
+  )
+}
+
+interface PagesWidgetProps {
+  site?: string;
+  path?: string;
+  locale?: string;
+}
+
+export default function PagesWidget(props: PagesWidgetProps) {
+  const classes = useStyles({});
+  const { formatMessage } = useIntl();
+  const { site = 'editorial', path = 'site/website', locale = 'en' } = props;
+  const [currentLocale, setCurrentLocale] = React.useState(locale);
+  const [selectMode, setSelectMode] = React.useState(false);
+  const [items, setItems] = useState<Item[]>(null);
+  const [breadcrumb, setBreadcrumb] = useState<Breadcrumb[]>([
+    {
+      id: 'Home',
+      label: 'Home',
+      path: 'home'
+    }
```

Comment:
This should also come from the API response. Re-check the API specs for updates.

Reply:
the api is not returning any breadcrumb

Reply:
Not breadcrumb but parent item is returned too.